src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/notification/EmailNotificationAsserter.kt

```diff
@@ -12,7 +12,7 @@ class EmailNotificationAsserter {
   private val requestedEmails = mutableListOf<EmailRequest>()
 
   @EventListener
-  fun emailRequested(emailRequested: SendEmailRequestedEvent) {
+  fun consumeEmailRequestedEvent(emailRequested: SendEmailRequestedEvent) {
```

Comment:
where are we using this function?

Reply:
This is called by spring when an event of type 'SendEmailRequesedEvent' is raised. I tried to rename it to better reflect that but still not sure it's 100% clear